src/components/RangeInput.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+<div v-if="state" :class="suit()">
+  <form :class="suit('form')" @submit.prevent="refine()">
+    <label :class="suit('label')">
+      <input :class="[suit('input'), suit('input', 'min')]" type="number" ref="minInput" :min="this.min" :placeholder="this.min"/>
```

Comment:
We should put the `min` & `max` restrictions on both input

Reply:
Does it mean that we should have dynamic setting based on the value of the other input?

Reply:
It's not mandatory but we can at least provide the range to min and max. Both inputs will be restricted on the range rather than only have the restriction on min or max.